mempool/src/shared_mempool/peer_manager.rs

```diff
@@ -187,6 +188,8 @@ impl PeerManager {
         }
 
         // When not a validator, only broadcast to `default_failovers`
+        let mut transmission_filter = None;
+
         if !self.role.is_validator() {
```

Comment:
oh I had hoped we were going to clean up this janky logic

Reply:
I had to add the jankiest part back in for cluster test